### PR TITLE
Personal Portfolio

### PR DESCRIPTION
**Pull request by danieldouglas1**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,15 +4,190 @@
...
+
+    <div id="header">
+        <h2>Daniel Douglas</h2>
     </div>
+
+    <div id= "section">
+        <div id="nav">
+            <ul>
+            <li><a href="#About Me">About Me</a></li>
+            <p> </p>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: I don't think this line is necessary. the `<ul>` and `<li>` elements form a list and the browser will automatically add new lines for each element in your list. You should just delete the lines with `<p> </p>` :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,15 +4,190 @@
...
+             <li><a href="#Favorite Movies">Favorite Movies</a></li>
+            <p> </p>
+            <li><a href="#Favorite Shows">Favourite Shows</a></li>
+            <p> </p>           
+            <li><a href="#Favorite Artistes">Favorite Artistes</a></li>
+            <p> </p>
+            <li><a href="#My Pictures">My Pictures</a></li>
+            <p> </p>
+            <li><a href="#">My Projects</a></li>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html:24]: I think this should be `<li><a href="#My Project">My Projects</a></li>`

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,15 +4,190 @@
...
+            <p> </p>
+            <li><a href="#">My Resume</a></li>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html:25]: I think this should be `<li><a href="#My Resume">My Resume</a></li>`

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -4,15 +4,190 @@
...
+
+   <div id="sections">
+    <h1><a name="About Me" </a> About Me </h1>
+    <p>Hi Everyone! My name is Daniel and I am a sophmore majoring in Computer Science at the College of the Holy Cross in Worcester, Massachusetts. I was born and raised in Kingston Jamaica and
+    enjoy listening to music, watching sports, playing soccer. Below you can find some of my favorite Movies, TV shows and Artistes.
+           
+    </p>
+   </div>
+
+    <! Favourite Movies >
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: Are you trying to write comments? If so, awesome! However, that's not the correct syntax. It should be `<!-- Favorite Movies -->`. More info at https://www.w3schools.com/html/html_comments.asp.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -4,15 +4,190 @@
...
     </div>
+
+    <div id= "section">
+        <div id="nav">
+            <ul>
+            <li><a href="#About Me">About Me</a></li>
+            <p> </p>
+             <li><a href="#Favorite Movies">Favorite Movies</a></li>
+            <p> </p>
+            <li><a href="#Favorite Shows">Favourite Shows</a></li>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: spelling: Favorite

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -4,15 +4,190 @@
...
+    <div class="gallery">
+        <a target="_blank" href="us.jpg">
+            <img src="/images/us.jpg" alt="US" width="600" height="400">
+        </a>
+    <div class="desc">Add a description of the image here</div>
+    </div>
+
+    </div>
+
+  <! Favorite Shows >
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: If this is a comment, then you should use a different syntax: https://www.w3schools.com/html/html_comments.asp

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -4,15 +4,190 @@
...
+            <li><a href="#">My Projects</a></li>
+            <p> </p>
+            <li><a href="#">My Resume</a></li>
+            </ul>
+        </div>
+    </div>
+  
+
+   <div id="sections">
+    <h1><a name="About Me" </a> About Me </h1>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This should be `<h1><a name="About Me"> About Me </a></h1>`. In HTML, you start a new element like `<a name="About Me">`, and then write whatever content you want. When you want to close your element, you add `</a>` at the end. Does that make sense? Writing HTML is a little confusing!

Comment (atreil) [reply portfolio/src/main/webapp/index.html]: You've made the same mistake in several different places below - I'll point them out to you.

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -4,15 +4,190 @@
...
+    <p>Hi Everyone! My name is Daniel and I am a sophmore majoring in Computer Science at the College of the Holy Cross in Worcester, Massachusetts. I was born and raised in Kingston Jamaica and
+    enjoy listening to music, watching sports, playing soccer. Below you can find some of my favorite Movies, TV shows and Artistes.
+           
+    </p>
+   </div>
+
+    <! Favourite Movies >
+   <div id="sections">
+    <h1><a name="Favorite Movies"</a>Favorite Movies(2019)</h1>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This should be `<h1><a name="Favorite Movies" >Favorite Movies(2019)</a></h1>`

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -4,15 +4,190 @@
...
+
+    <div class="gallery">
+        <a target="_blank" href="1917.jpg">
+            <img src="/images/1917.jpg" alt="1917" width="600" height="400">
+        </a>
+    <div class="desc">Add a description of the image here</div>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: I'm assuming you'll add a description later? :)

File: portfolio/src/main/webapp/index.html [hunk 10]
```diff
@@ -4,15 +4,190 @@
...
+    </p>
+   </div>
+
+    <! Favourite Movies >
+   <div id="sections">
+    <h1><a name="Favorite Movies"</a>Favorite Movies(2019)</h1>
+
+    <div class="gallery">
+        <a target="_blank" href="1917.jpg">
+            <img src="/images/1917.jpg" alt="1917" width="600" height="400">
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This is close to correct! In HTML there are two ways of writing elements. The first way is to have open and closing tags, like: ` </img>`. The `</img>` is a closing tag. That can be pretty verbose, so you can write it as `` instead. This is called a void tag. In your case, you almost have a void tag but you need to add an extra `/` just before the last `>` :)

Comment (atreil) [reply portfolio/src/main/webapp/index.html]: You make the same mistake several times below where you link an image, so I won't comment on each one but you should fix those ones as well.

File: portfolio/src/main/webapp/index.html [hunk 11]
```diff
@@ -4,15 +4,190 @@
...
+            <img src="/images/us.jpg" alt="US" width="600" height="400">
+        </a>
+    <div class="desc">Add a description of the image here</div>
+    </div>
+
+    </div>
+
+  <! Favorite Shows >
+  <div id="sections">
+    <h1><a name="Favorite Shows"</a>Favorite Shows</h1>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This should be `<h1><a name="Favorite Shows" >Favorite Shows</a></h1>`

File: portfolio/src/main/webapp/index.html [hunk 12]
```diff
@@ -4,15 +4,190 @@
...
+        <a target="_blank" href="/images/you.jpeg">
+            <img src="/images/you.jpeg" alt="You" width="600" height="400">
+        </a>
+    <div class="desc">Add a description of the image here</div>
+    </div>
+
+  </div>
+ 
+  <div id="sections">
+    <h1><a name="Favorite Artistes" </a>Favorite Artistes</h1>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This should be `<h1><a name="Favorite Artistes">Favorite Artistes </a></h1>`

File: portfolio/src/main/webapp/index.html [hunk 13]
```diff
@@ -4,15 +4,190 @@
...
+    <h1><a name= "My Pictures" </a> My Pictures</h1>
+    <div class="gallery">
+        <a target="_blank" href="/images/image1.JPG">
+            <img src="/images/image1.JPG" alt="You" width="600" height="400">
+        </a>
+    </div>
+
+  
+  </div>
+  <! Footer >
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: You should use the proper commenting syntax: https://www.w3schools.com/html/html_comments.asp

File: portfolio/src/main/webapp/index.html [hunk 14]
```diff
@@ -4,15 +4,190 @@
...
+    <div id="footer">
+     <p>Connect With Me</p>
+     <a href="https://github.com/danieldouglas1">
+        <i class="fab fa-github"style="font-size:24px;color:white;"></i>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: This is actually really cool! I didn't know you could do this. However, if someone is using a white background (like I am) then you can't actually see the icons. Try changing the color from white to black. This comment also applies below to the linkedin block.

**Review by atreil — APPROVED** [review]
Overall, this is a really good start! Writing HTML can be difficult so good job! I left several comments, mostly related to syntax.

You can merge in this pull request, and then send me another pull request fixing the issues that I have pointed out.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (danieldouglas1) [reply portfolio/src/main/webapp/index.html:25]: I was planning to add an external link to my resume

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (danieldouglas1) [reply portfolio/src/main/webapp/index.html]: I sometimes forget and still use the European spelling :)

**Pull request merged**